packages/channel-client/tests/fakes/fake-channel-provider.ts

```diff
@@ -34,19 +34,30 @@ type ChannelId = string;
  coming from a non-fake `ChannelClient`.
  */
 export class FakeChannelProvider implements ChannelProviderInterface {
+  public address?: string;
+  public selectedAddress?: string;
+  public walletVersion?: string;
+
   private events = new EventEmitter<EventType>();
   protected url = '';
 
   playerIndex: Record<ChannelId, 0 | 1> = {};
   opponentIndex: Record<ChannelId, 0 | 1> = {};
-  address: string = Wallet.createRandom().address;
+  internalAddress: string = Wallet.createRandom().address;
```

Comment:
We are using `signingAddress` elsewhere, and I'm mindful that we don't want to proliferate too many names for the same thing. 

To further muddy the waters / potentially clarify the terminology, @kerzhner made the point that we could call this `channelPublicKey` to reflect its limited role in the context in which we use it, and help disambiguate from other things like `outcomeAddress`  / `ethereumAddress`.
https://statechannels.slack.com/archives/CQJCSTKA7/p1584026202014600

Perhaps best to stick with `signingAddress` for this PR, and then consider a global change of terminology in a separate one?

Reply:
Agree, see https://github.com/statechannels/monorepo/pull/1281/commits/db5b4067a32b83bb6a4d18e2abe95926a677e275

Reply:
I think some instances of `internalAddress` still snuck in?